### DISCUSSION TITLE
Support for KMS

### DISCUSSION THREAD

**Discussion by bharatnpti** [awspring/spring-cloud-aws#860] [Ideas]
Do we have any plans to support AWS KMS as well.
It will help to auto configure things like token with AWS STS, similar to what has been done with AWS Secret Manager to Assume Role

**Comment by MatejNedic** [comment]
Hey @bharatnpti , could you please provide what would you expect from the integration and open it as an Issue.

**Reply by bharatnpti** [reply]
Hi @MatejNedic - The required functionality has already been added with this PR - https://github.com/awspring/spring-cloud-aws/pull/691

**Comment by bharatnpti** [comment]
The required functionality has already been added with this PR - https://github.com/awspring/spring-cloud-aws/pull/691
